sympy/stats/tests/test_continuous_rv.py

```diff
@@ -726,6 +726,41 @@ def test_precomputed_cdf():
         compdiff = simplify(compdiff.rewrite(erfc))
         assert compdiff == 0
 
+
+def test_precomputed_characteristic_functions():
+    import mpmath
+
+    def test_cf(dist, test_point, support_lower_limit, support_upper_limit):
+        pdf = density(dist)
+        t = Symbol('t')
+        x = Symbol('x')
+
+        # first function is the hardcoded CF of the distribution
+        cf1 = lambdify([t], characteristic_function(dist)(t), 'mpmath')
+
+        # second function is the Fourier transform of the density function
+        f = lambdify([x,t], pdf(x)*exp(I*x*t), 'mpmath')
```

Comment:
missing space after comma here and on the following line (would be caught by pep8online.com)